tools/makeboards.py

```diff
@@ -39,7 +39,7 @@ def BuildDebugLevel(name):
 
 def BuildFreq(name, defmhz):
     out = 0
-    for f in [ defmhz, 50, 100, 120, 125, 128, 133, 150, 175, 200, 225, 240, 250, 275, 300]:
+    for f in [ defmhz, 50, 100, 120, 125, 128, 133, 150, 153.6, 175, 200, 225, 240, 250, 275, 300]:
```

Comment:
This requires re-running the script and checking in the updated boards.txt or CI will fail.